src/workspaces/list/RenderedWorkspaces.test.ts

```diff
@@ -79,15 +125,11 @@ describe('The behavior of the RenderedWorkspaces component', () => {
     render(h(RenderedWorkspaces, { workspaces: [workspace], label, noContent: div({}) }));
 
     // Assert
-    const workspaceDescriptionDisplay = screen.queryAllByText('some description');
-    expect(workspaceDescriptionDisplay).toHaveLength(0);
-
-    const workspaceStateDisplay = screen.getAllByText('Error deleting workspace');
-    expect(workspaceStateDisplay).not.toBeNull();
-    expect(workspaceStateDisplay).toHaveLength(1);
+    const workspaceNameLink = screen.getAllByText(defaultAzureWorkspace.workspace.name);
+    expect(workspaceNameLink[0]).toHaveAttribute('aria-disabled', linkDisabled.toString());
```

Comment:
Note that you can also get a link by the aria role.

For instance, `const codeAndWorkflows = await screen.getByRole('link', { name: 'code & workflows' });`